### ISSUE TITLE
Grunt tasks should honour workDir property

### ISSUE THREAD

**Issue by Antillean** [srs/gradle-grunt-plugin#22]
There doesn't seem to be a way of setting Grunt's working directory. Grunt install always seems to install grunt to ```node_modules/grunt``` and ```node_modules/grunt-cli``` in my project directory, and it also always looks for Gruntfile.js in my project directory. I made a custom install task as follows to get more generic behaviour:

```javascript
task gruntInstallCustom(type: NpmTask, dependsOn: npmInstallCustom) {
    execOverrides {
        it.workingDir = nodeBuildDir
    }

    setArgs( ['install', 'grunt-cli', 'grunt'] )

    outputs.dir { file("$nodeBuildDir/node_modules/grunt") }
    outputs.dir { file("$nodeBuildDir/node_modules/grunt-cli") }
}
```

It might be worth making a similar/corresponding change in the default grunt tasks. 

(This corresponds to [issue 48 of gradle-node-plugin](https://github.com/srs/gradle-node-plugin/issues/48). npmInstallCustom is defined on that ticket.)

**Comment by btiernay** [comment]
Was this fixed? I'm still having the same issue. For my grunt tasks, I would like to be able to use the same npm modules directory configured using `node`. Thanks!

**Comment by btiernay** [comment]
I think this is still an issue due to:

https://github.com/srs/gradle-grunt-plugin/blob/master/src/main/groovy/com/moowork/gradle/grunt/GruntTask.groovy#L19

Essentially, this binds grunt binaries relative to the project directory. This should be done relative to the working directory. Otherwise, it is required that `"${project.projectDir}/node_modules/"` exist, which limits customization / flexibility.

**Comment by btiernay** [comment]
@Antillean I wasn't able to successfully use your custom task suggestion above due to the binding of `GruntTask` to the project directory. Although grunt will get installed to your specified folder, it still seems that the grunt tasks fail if you remove `"${project.projectDir}/node_modules/"`

**Comment by Antillean** [comment]
@btiernay, you're right. I forgot to include another task redefinition that I'm using to get it to work:
```groovy
task gruntBuild(type: NodeTask, dependsOn: gruntInstallCustom) {
    execOverrides {
        it.workingDir = nodeBuildDir
    }

    setScript(file("$nodeBuildDir/node_modules/grunt-cli/bin/grunt"))
}
```
And the setScript overrides the problematic line that you pointed out.
